terraform/globals.tf

```diff
@@ -57,13 +57,13 @@ variable "vagrant_cidr" {
 }
 
 variable "dns_zone_id" {
-  description = "Amazon Route53 DNS zone identifier"
-  default     = "Z3SI0PSH6KKVH4"
+  description = "Amazon Route53 DNS zone identifier. Different per account."
+  default     = "dummy_default"
 }
 
 variable "dns_zone_name" {
-  description = "Amazon Route53 DNS zone name"
-  default     = "cf.paas.alphagov.co.uk"
+  description = "Amazon Route53 DNS zone name. Differs per account."
+  default     = "dummy_default"
```

Comment:
These values are safe dummies that will break TF deployment if they leak. When you run TF with limited scope, without defaults it would still ask you for the value of these variables regardless if it needs to use them. TF will update some related outputs (and can lose some, see [this](https://github.com/alphagov/paas-cf/commit/164f954789c5e53b9e8c47a4e6cd758f4451e95d)) when running on limited scope.